app/scripts/metamask-controller.js

```diff
@@ -149,6 +149,10 @@ import {
   getLocalizedSnapManifest,
   stripSnapPrefix,
 } from '@metamask/snaps-utils';
+import {
+  methodsRequiringNetworkSwitch,
+  methodsWithConfirmation,
+} from '../../shared/constants/methods-tags';
 ///: END:ONLY_INCLUDE_IF
```

Comment:
seems like this shouldn't be in the `ONLY_INCLUDE_IF(keyring-snaps)` block

Reply:
thanks! I was sure I pasted it else where.. guess not woops!

Reply:
errrr yeah so I guess I was correct to be sure... something wrong with our lint:fix:
https://github.com/MetaMask/metamask-extension/pull/24061/commits/20f03221195b4f42c348f4717d862899fed6c51d

Reply:
lol more like `yarn lint:break` am i right?